pix2tex/dataset/render.py

```diff
@@ -7,112 +8,165 @@
 from tqdm.auto import tqdm
 import cv2
 import numpy as np
+from PIL import Image
+import traceback
+import subprocess
+import shlex
 
 
-def render_dataset(dataset: np.ndarray, names: np.ndarray, args):
-    '''Renders a list of tex equations
+def get_installed_fonts(tex_path: str):
+    cmd = "find %s -name *Math*.otf" % tex_path
+    process = subprocess.Popen(shlex.split(cmd),
+                               stdout=subprocess.PIPE,
+                               stderr=subprocess.PIPE,
+                               universal_newlines=True
+                               )
+    stdout, stderr = process.communicate()
+    if process.returncode != 0:
+        raise Exception(stderr)
+    fonts = [_.split(os.sep)[-1] for _ in stdout.split('\n')][:-1]
+    fonts.append("Latin Modern Math")
```

Comment:
That makes LM as likely as the other fonts. I think it's important that this is at least as likely as all non standard fonts. 
Maybe change to
```python
fonts.extend(["Latin Modern Math"]*len(fonts))
```